bin/sstableloader

```diff
@@ -32,18 +35,6 @@ elif [ -r "$CASSANDRA_INCLUDE" ]; then
     . "$CASSANDRA_INCLUDE"
 fi
 
-if [ -z "$CLASSPATH" ]; then
-    echo "You must set the CLASSPATH var" >&2
-    exit 1
-fi
-
-if [ "x$MAX_HEAP_SIZE" = "x" ]; then
-    MAX_HEAP_SIZE="256M"
-fi
-
-"$JAVA" $JAVA_AGENT -ea -cp "$CLASSPATH" $JVM_OPTS -Xmx$MAX_HEAP_SIZE \
-        -Dcassandra.storagedir="$cassandra_storagedir" \
-        -Dlogback.configurationFile=logback-tools.xml \
-        org.apache.cassandra.tools.BulkLoader "$@"
+"$CASSANDRA_HOME/tools/bin/sstableloader" "$@"
```

Comment:
@tolbertam  I would check the existence of this script because sstableloader is a tool which is / should be installed by a different package (redhat/cassandra.spec).

If it is not present, then we need to tell a user that they should install tools package.

Reply:
I suppose now that we're bundling sstableloader with tools, using it will now require the cassandra-tools package, where previously it was bundled with cassandra.  I hope this is the kind of change we can make between 5.0 and whatever the next release?

We could in theory still bundle sstableloader with the core package, it's just that it will ultimately its library will live in the tools directory.

I'll go with bundling it in the tools package and updating this scrip to check for presence of the script and erroring out with a message as you suggest.  I will also add something to news for this.

Reply:
actually, its interesting;  It looks like both fqltool and cassandra-stress are packaged in the main cassandra package, not the tools package, despite the package designation.  This is a bit contrary to what I expected, but other tools like sstabledump, which don't have their own tools directory/library path are in the tools package.

I think what I should do is just retain that behavior and keep sstableloader packaged in the cassandra package.

Reply:
Findings:

debian:  `sstableloader` was already only bundled in cassandra-tools package, where `fqltool` and `cassandra-stress` were in the base cassandra_all

redhat: `sstableloader` is bundled in the primary cassandra package.

I think given the inconsistency, it's probably not best to fuss with this, and instead of focus on making it work as is.  Will push some changes shortly that accomplishes this.

Reply:
@tolbertam I see. So the plan is to keep packaging sstableloader as is, it would be just a different class path etc. Right?

Reply:
That's right,  I think given stress and fqltool are bundled in the main package already (even though they have separate builds into tools), we might as well do the same for sstableloader.